ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/query/BlockchainQueries.java

```diff
@@ -820,7 +820,10 @@ private TransactionWithMetadata transactionByHeaderAndIndex(
                         .orElseThrow(
                             () -> new IllegalStateException("Could not retrieve block #" + l)))
             .flatMap(Collection::stream)
-            .mapToLong(t -> t.getGasPrice().toLong())
+            // TODO: currently we omit 1559 transactions from this calculation, what should
+            //       gasPrice() return for this endpoint post-London? (substitute effective gas?)
```

Comment:
Just a sanity check. Was this TODO supposed to be here? I mean, did we forget to do something or is this a reminder for a future change?

Reply:
this is a reminder I forgot to remove - it is added to #2441.  Thanks for catching